core/src/main/java/com/google/ar/sceneform/rendering/MaterialParameters.java

```diff
@@ -252,6 +268,16 @@ void applyTo(MaterialInstance materialInstance) {
     }
   }
 
+  void applyParameterTo(MaterialInstance materialInstance, String name) {
+    com.google.android.filament.Material material = materialInstance.getMaterial();
+
+    if(material.hasParameter(name)) {
+      Optional
```

Comment:
Maybe it is better to use `if` in Java?

Reply:
Optional is available since Java 8. To be fair, it is a personal thing of mine to use Optional. But I have no problem to replace it with an if statement.